invenio_rdm_records/theme/templates/invenio_rdm_records/details/contributors.html

```diff
@@ -0,0 +1,21 @@
+{#
+    Copyright (C) 2020 CERN.
+    Copyright (C) 2020 Northwestern University.
+  
+    Invenio RDM Records is free software; you can redistribute it and/or modify
+    it under the terms of the MIT License; see LICENSE file for more details.
+#}
+
+{%- for group in record.contributors|groupby('role')%}
+<h5><b>{{group.grouper|contributortype_title}}(s)</b></h5>
+
+{%- for creator in group.list %}
+{%- if creator.identifiers[0].scheme == "Orcid" %}
+<a href="{{creator.identifiers[0].identifier|pid_url('orcid')}}">
+    <img class="inline-orcid" src="{{ url_for('static', filename='images/orcid.png')}}" />
+</a>
+{%- endif %}
+<span class="text-muted" {% if creator.affiliations and creator.affiliations[0] %}data-toggle="tooltip" title="{{creator.affiliations[0].name}}"{% endif %}>{{creator.name}}</span>{% if not loop.last %}; {% endif %}
```

Comment:
future/custom: could eventually get away with no `;` and always have an icon (even if generic) next to a creator

Reply:
https://github.com/inveniosoftware/invenio-rdm-records/issues/61